### DISCUSSION TITLE
Custom values to be changed in examples/minikube/custom_values.yaml

### DISCUSSION THREAD

**Discussion by Adaverse** [airflow-helm/charts#644] [Questions & Answers]
Hello all,

Was trying to deploy the Airflow using Helm with some modification in custom_values.yaml using minikube cluster. But many pods end up in error. Does anyone know what changes needs to be done in the custom_values.yaml to make the deployment successful.

Thank you in advance.

**Comment by thesuperzapper** [comment]
We will need more information to help you, please share your custom values and error logs.
